UPGRADING.md

```diff
@@ -38,10 +38,30 @@ validator and ABCI
 built from `ResponseDeliverTx(Code, Data)` responses, became the root hash of a
 Merkle tree built from:
 
-  - `BeginBlock#Events`;
-  - root hash of a Merkle tree built from `ResponseDeliverTx(Code, Data,
+- `BeginBlock#Events`;
+- root hash of a Merkle tree built from `ResponseDeliverTx(Code, Data,
     GasWanted, GasUsed, Events)` responses;
-  - `BeginBlock#Events`.
+- `BeginBlock#Events`.
+
+### Tx Indexing
+
+- Tendermint will now rely on the application entirely to tell it what txs to index. This means that in the `config.toml`,
```

Comment:
Instead of making this a bullet point list, I would remove the 2nd bullet point item and just include it here as sentence.